### PR TITLE
#37- Retroactively applying linter (black, isort) + Lint on CI

### PR DESCRIPTION
**Pull request by pfaraone**
# Issue
#37

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -35,4 +35,6 @@ ignore =
   B015
   B016
   B017
+  F541
```
Comment (pfaraone) [inline .flake8:38]: ignore 
- f-string without placeholder

**Review by pdames — APPROVED** [review]
LGTM!

**Review by valiantljk — APPROVED** [review]
LGTM, thanks.

**Review by raghumdani — COMMENTED** [review]
I assume there is no manual changes after or before applying linter, however `README about how to run linter and integrate it with vscode or pycharm` would be helpful for other maintainers.

**Review by raghumdani — APPROVED** [review]

**Comment by pdames** [comment]
To ensure that styling remains consistent across subsequent PRs, please provide a simple script that can be run to setup Git commit hooks. For example:

https://github.com/ray-project/ray/blob/master/setup_hooks.sh

We should also ensure that our CI workflow runs a linter check on all PRs. For example, here's a similar check that ensures that Apache license header is present in all new files added to a PR:

https://github.com/ray-project/ray_beam_runner/blob/master/.github/workflows/ci.yml#L74-L85

**Comment by pfaraone** [comment]
> To ensure that styling remains consistent across subsequent PRs, please provide a simple script that can be run to setup Git commit hooks. For example:
> 
> https://github.com/ray-project/ray/blob/master/setup_hooks.sh
> 
> We should also ensure that our CI workflow runs a linter check on all PRs. For example, here's a similar check that ensures that Apache license header is present in all new files added to a PR:
> 
> https://github.com/ray-project/ray_beam_runner/blob/master/.github/workflows/ci.yml#L74-L85

CI workflow now runs a linter check on all PRs against main.

> To ensure that styling remains consistent across subsequent PRs, please provide a simple script that can be run to setup Git commit hooks. For example:

I don't think this is necessary at the moment as pre-commit comes with all the batteries included to set itself and the linters up  (just run `pre-commit install` - https://pre-commit.com/#3-install-the-git-hook-scripts) . I could add to a README-dev.md that you have to `pip install dev-requirements.txt` then `pre-commit install` to run pre-commit locally

File: setup.py [hunk 2]
```diff
@@ -25,7 +25,7 @@ def find_version(*paths):
 
 
 setuptools.setup(
-    name="deltacat",
+    name="deltacat-fork",
```
Comment (pdames) [inline setup.py:28]:
```suggestion
    name="deltacat",
```

**Review by pdames — APPROVED** [review]
LGTM! I applied a quick fix to revert the test package name left in `setup.py`

**Pull request merged**